### PR TITLE
Fixes #13009: Documentation for API authorizations plugin

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/13009

### CODE REVIEW

**Comment by fanf** [comment]
OK, merging this PR

File: api-authorizations/README.adoc [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+   -->
+  <logger name="api-acl" level="off" />
+```
+
+// Everything after this line goes into Rudder documentation
+// ====doc====
+
+
+=== User personnal API Token
```
Comment (amousset) [inline api-authorizations/README.adoc]: This should be a first level title "=", doc level integration is handle by the doc.

File: api-authorizations/README.adoc [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+When you use `api-authorizations` plugin, any logged user can get a personnal API token by clicking on
+its loggin information:
```
Comment (amousset) [inline api-authorizations/README.adoc]: login

File: api-authorizations/README.adoc [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      This logger allows to get extra information about API
+      ACLs resolution.
+
+   -->
+  <logger name="api-acl" level="off" />
+```
+
+// Everything after this line goes into Rudder documentation
+// ====doc====
+
```
Comment (amousset) [inline api-authorizations/README.adoc]: We need a first level title here with the plugin name

**Pull request merged**